v20.1/backup-and-restore-advanced-options.md

```diff
@@ -0,0 +1,334 @@
+The list of URIs passed to [`RESTORE`][restore] may be different from the URIs originally passed to [`BACKUP`][backup]. This is because the files of a locality-aware backup can be moved to different locations, or even consolidated into the same location. The only restriction is that all the files originally written to the same location must remain together. In order for [`RESTORE`][restore] to succeed, all of the files originally written during [`BACKUP`][backup] must be accounted for in the list of location URIs provided.
+{{site.data.alerts.end}}
+
+### Create a locality-aware backup
+
+For example, to create a locality-aware backup where nodes with the locality `region=us-west` write backup files to `s3://us-west-bucket`, and all other nodes write to `s3://us-east-bucket` by default, run:
+
+{% include copy-clipboard.html %}
+~~~ sql
+> BACKUP TO
+	  ('s3://us-east-bucket?COCKROACH_LOCALITY=default', 's3://us-west-bucket?COCKROACH_LOCALITY=region%3Dus-west');
+~~~
+
+can be restored by running:
+
+{% include copy-clipboard.html %}
+~~~ sql
+> RESTORE FROM ('s3://us-east-bucket', 's3://us-west-bucket');
+~~~
+
+Note that the first URI in the list has to be the URI specified as the `default` URI when the backup was created. If you have moved your backups to a different location since the backup was originally taken, the first URI must be the new location of the files originally written to the `default` location.
+
+### Restore from a locality-aware backup
+
+You can create locality-aware backups such that each node writes files only to the backup destination that matches the [node locality](configure-replication-zones.html#descriptive-attributes-assigned-to-nodes) configured at [node startup](cockroach-start.html).
+
+A locality-aware backup is specified by a list of URIs, each of which has a `COCKROACH_LOCALITY` URL parameter whose single value is either `default` or a single locality key-value pair such as `region=us-east`. At least one `COCKROACH_LOCALITY` must be the `default`. Given a list of URIs that together contain the locations of all of the files for a single locality-aware backup, [`RESTORE`][restore] can read in that backup.
+
+Note that the list of URIs passed to [`RESTORE`][restore] may be different from the URIs originally passed to [`BACKUP`][backup]. This is because it's possible to move the contents of one of the parts of a locality-aware backup (i.e., the files written to that destination) to a different location, or even to consolidate all the files for a locality-aware backup into a single location.
+
+{{site.data.alerts.callout_info}}
+[`RESTORE`][restore] is not truly locality-aware; while restoring from backups, a node may read from a store that does not match its locality. This can happen because [`BACKUP`][backup] does not back up [zone configurations](configure-replication-zones.html), so [`RESTORE`][restore] has no way of knowing how to take node localities into account when restoring data from a backup.
```

Comment:
`his can happen because [`BACKUP`][backup] does not back up [zone configurations](configure-replication-zones.html)`

This particular sentence is different with full cluster backup, where we do store system.zones. On restore though, it is true that we don't process files necessarily on the zone specific node, so the second sentence is true. It's a bit tricky how we would update this. @dt @pbardea thoughts?

Reply:
Hmm, I would probably suggest something along the lines of:
> This can happen in the cases that either the `BACKUP` or `RESTORE` was not full-cluster. Note that during a locality-aware restore, some data may be temporarily located on another node before it is eventually re-located to the appropriate node.

As a note to the second sentence, we do have a work-around of:
1. Pausing the restore
2. Manually restoring the zone configs
3. Resume the restore
to prevent the rebalancing. I'm not sure what the best place is to document this?